packages/shared/src/contexts/FeaturesContext.tsx

```diff
@@ -47,41 +31,10 @@ export interface FeaturesContextProviderProps
 }
 
 const getFeatures = (flags: IFlags): FeaturesData => {
-  const steps = getFeatureValue(Features.OnboardingSteps, flags);
-  const onboardingSteps = (steps?.split?.('/') || []) as OnboardingStep[];
-  const minimumTopics = getFeatureValue(
-    Features.OnboardingMinimumTopics,
-    flags,
-  );
-
   return {
     flags,
-    onboardingSteps,
-    onboardingMinimumTopics: getNumberValue(minimumTopics, 0),
-    onboardingFiltersLayout: getFeatureValue(
-      Features.OnboardingFiltersLayout,
-      flags,
-    ),
-    popularFeedCopy: getFeatureValue(Features.PopularFeedCopy, flags),
-    showCommentPopover: isFeaturedEnabled(Features.ShowCommentPopover, flags),
     canSubmitArticle: isFeaturedEnabled(Features.SubmitArticle, flags),
-    submitArticleSidebarButton: getFeatureValue(
-      Features.SubmitArticleSidebarButton,
-      flags,
-    ),
-    submitArticleModalButton: getFeatureValue(
-      Features.SubmitArticleModalButton,
-      flags,
-    ),
-    inAppNotificationPosition: getFeatureValue(
-      Features.InAppNotificationPosition,
-      flags,
-    ),
     onboardingV2: getFeatureValue(Features.OnboardingV2, flags),
-    onboardingFilteringTitle: getFeatureValue(
-      Features.OnboardingFilteringTitle,
-      flags,
-    ),
```

Comment:
This has yet to be used. Sab decided to use this a few weeks after v2. I think we should keep this.

Reply:
Oh valid point, actually missed that 😅